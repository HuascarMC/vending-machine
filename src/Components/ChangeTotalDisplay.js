import React from 'react';

class ChangeTotalDisplay extends React.Component {
  render() {
    return(
      <div className="display">
        <p className="balance">$0.00</p>
      </div>
    );
  }
}

export default ChangeTotalDisplay;
