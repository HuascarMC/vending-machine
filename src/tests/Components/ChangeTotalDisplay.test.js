import React from 'react';
import ChangeTotalDisplay from '../../Components/ChangeTotalDisplay'
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';

it('should render without crashing', () => {
  shallow(<ChangeTotalDisplay amount={0.00}/>);
});

it('should match the snapshot', () => {
  const component = renderer.create(<ChangeTotalDisplay amount={0.00}/>);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
