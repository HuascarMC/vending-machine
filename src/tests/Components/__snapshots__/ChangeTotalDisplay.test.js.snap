// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match the snapshot 1`] = `
<div
  className="display change-display"
>
  <p
    className="display-text"
  >
    $
    0
  </p>
</div>
`;
